packages/plugin-backgroundcolor/src/BackgroundColor.ts

```diff
@@ -0,0 +1,164 @@
+import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
+import { CommandParams } from '../../core/src/Dispatcher';
+import { VNode } from '../../core/src/VNodes/VNode';
+import { Loadables } from '../../core/src/JWEditor';
+import { Keymap } from '../../plugin-keymap/src/Keymap';
+import { Inline } from '../../plugin-inline/src/Inline';
+import { InlineNode } from '../../plugin-inline/src/InlineNode';
+import { getStyles, setStyles, setStyle, getStyle, removeStyle } from '../../utils/src/utils';
+import { VRange } from '../../core/src/VRange';
+import { Format } from '../../plugin-inline/src/Format';
+
+export interface ColorParams extends CommandParams {
+    color?: string; // css-valid color name
+}
+
+interface BackgroundColorConfig extends JWPluginConfig {
+    defaultBackgroundColor?: string;
+}
+
+const defaultConfiguration = {
+    defaultBackgroundColor: 'white',
+};
+
+export class BackgroundColor<T extends BackgroundColorConfig = BackgroundColorConfig> extends JWPlugin<T> {
+    static hasColor(node: VNode): boolean;
+    static hasColor(backgroundColor: string, node: VNode): boolean;
+    static hasColor(backgroundColor: string | VNode, node?: VNode): boolean {
```

Comment:
If you renamed `hasBackgroundColor` into `hasColor` then why not also renaming the parameter ? It's the static method of `BackgroundColor`, of course `color` is going to refer to the background color... ;-)